logfire/_internal/exporters/processor_wrapper.py

```diff
@@ -47,13 +48,7 @@ def on_end(self, span: ReadableSpan) -> None:
         _tweak_http_spans(span_dict)
         self.scrubber.scrub_span(span_dict)
         span = ReadableSpan(**span_dict)
-        self.processor.on_end(span)
-
-    def shutdown(self) -> None:
-        self.processor.shutdown()
-
-    def force_flush(self, timeout_millis: int = 30000) -> bool:
```

Comment:
is this the method we were using to get logfire to work with AWS lambda? If so, I guess we need it.

Reply:
It's moved to the base class.